tests/conftest.py

```diff
@@ -10,6 +10,8 @@
 load_dotenv(dotenv_path=DOTENV_PATH)
 
 # Import fixtures from our package so pytest can detect them
+
+
```

Comment:
remove these blank lines
shouldn't have blanks between the comment ("# Import...") and the affected code ("pytest_...")